apps/climatemappedafrica/src/components/HowItWorks/index.js

```diff
@@ -0,0 +1,192 @@
+              },
+              top: {
+                md: 0,
+              },
+              backgroundColor: {
+                lg: theme.palette.background.default,
+              },
+              opacity: {
+                md: 0.9,
+              },
+              height: {
+                md: theme.typography.pxToRem(524),
+                lg: theme.typography.pxToRem(600),
+              },
+              padding: {
+                md: `${theme.typography.pxToRem(66)} ${theme.typography.pxToRem(
+                  77,
+                )} ${theme.typography.pxToRem(69)} 0`,
+                lg: `${theme.typography.pxToRem(81)} ${theme.typography.pxToRem(98)}`,
+              },
+            })}
+          >
+            <Box
+              sx={(theme) => ({
+                position: "relative",
+                "& .video-js": {
+                  width: "100%",
+                  height: "100%",
+                },
+                "& .vjs-poster": {
+                  backgroundColor: "#ffffffE6",
+                  backgroundSize: "120%",
+                },
+                "& .video-js .vjs-big-play-button": {
+                  display: "none",
+                },
+                [theme.breakpoints.up("md")]: {
+                  "& .vjs-poster": {
+                    backgroundSize: "auto",
+                  },
+                },
+                height: {
+                  xs: theme.typography.pxToRem(227),
+                  md: theme.typography.pxToRem(194),
+                  lg: theme.typography.pxToRem(244),
+                },
+                width: {
+                  xs: "100%",
+                  md: theme.typography.pxToRem(299),
+                  lg: theme.typography.pxToRem(376),
+                },
+              })}
+            >
+              <Player {...video} />
+            </Box>
+            <Typography
+              sx={(theme) => ({
+                marginTop: theme.typography.pxToRem(18),
+              })}
+              variant="h4"
+            >
+              {title}
+            </Typography>
+            <RichText
+              elements={description}
+              sx={(theme) => ({
+                fontFamily: theme.typography.body1.fontFamily,
+                margin: `${theme.typography.pxToRem(16.5)} 0`,
+                color: theme.palette.grey.dark,
+                width: {
+                  md: theme.typography.pxToRem(278),
+                },
+              })}
+            />
+            <Button href={link.href} variant="text">
+              {link.label}
+            </Button>
+          </Grid>
+          <Grid item lg={1} />
+          <Grid
+            item
+            xs={12}
+            md={5}
+            sx={(theme) => ({
+              display: "flex",
+              alignItems: "center",
+              justifyContent: {
+                xs: "center",
+                md: "flex-end",
+              },
+              marginBottom: theme.typography.pxToRem(23),
```

Comment:
I'm still a bit iffy on styling `Grid item` like this; can't all of these styling be done in the `Box` inside the `Grid item`?